portfolio/src/main/java/com/google/sps/servlets/DataServlet.java

```diff
@@ -21,14 +21,21 @@
 import javax.servlet.http.HttpServlet;
 import javax.servlet.http.HttpServletRequest;
 import javax.servlet.http.HttpServletResponse;
+import com.google.gson.Gson;
 
 /** Servlet that returns some example content. TODO: modify this file to handle comments data */
 @WebServlet("/data")
 public class DataServlet extends HttpServlet {
     private final List<Country> countries = new ArrayList();
+    private final List<String> hardCodedJson = new ArrayList();
 
     @Override
     public void init() {
+        countryInit();
+        jsonInit();
+    }
+
+    public void countryInit() {
```

Comment:
this kind of internal initialization method (only makes sense to call this from the constructor) should be marked `private`